plugins/quickbars.md

```diff
@@ -59,61 +60,77 @@ tinymce.init({
 });
 ```
 
-### Plugin-specific controls
+#### Example disabling the Quick Image context toolbar:
+
+```js
+tinymce.init({
+    selector: "div.tinymce",
+    plugins: [ 'quickbars' ],
+    toolbar: false,
+    menubar: false,
+    inline: true,
+    quickbars_image_toolbar: false
+});
+```
+
+### Plugin-specific toolbar buttons
 
 #### Quick Link
 
-The Quick Link (`quicklink`) control lets the user quickly insert/edit links inline. It can only be used in the Quick Selection toolbar.
+The Quick Link (`quicklink`) toolbar button lets the user quickly insert/edit links inline. It is included in the Quick Selection context toolbar by default and can be used in other context toolbars.
 
 ##### Example:
 
 ```js
 tinymce.init({
     selector: "div.tinymce",
     plugins: [ 'quickbars' ],
-    toolbar: false,
-    menubar: false,
-    inline: true,
-    quickbars_selection_toolbar: 'bold italic | quicklink h2 h3 blockquote'
+    setup: function(editor) {
+        editor.ui.registry.addContextToolbar('imageselection', {
+            predicate: function(node) {
+                return node.nodeName === 'P';
+            },
+            items: 'quicklink',
+            position: 'node'
+        });
+    }
 });
 ```
 
 #### Quick Image
 
-The Quick Image (`quickimage`) control lets you quickly insert images from the local computer into the editor. These can then be automatically uploaded if you configure [image uploading]({{ site.baseurl }}/advanced/handle-async-image-uploads/). It can be used in both the Quick Insert toolbar and other toolbars.
+The Quick Image (`quickimage`) toolbar button helps you quickly insert images from your computer into the editor. It is included in the Quick Insert context toolbar by default and can be used in other toolbars.
 
-##### Example:
+> Note: To enable automatic upload of images on insertion, [image upload]({{ site.baseurl }}/advanced/handle-async-image-uploads/) must be configured.
+
+##### Example using quickimage in the editor toolbar
 
 ```js
 tinymce.init({
     selector: "div.tinymce",
     plugins: [ 'quickbars' ],
-    toolbar: false,
-    menubar: false,
-    inline: true,
-    quickbars_insert_toolbar: 'quickimage quicktable'
+    toolbar: 'quickimage'
 });
 ```
 
 #### Quick Table
 
-The Quick Table (`quicktable`) control lets you quickly insert a 2x2 table with 100% width. It can be used in both the Quick Insert toolbar and other toolbars.
+The Quick Table (`quicktable`) toolbar button inserts a 2x2 table with 100% width. It is included in the Quick Insert context toolbar by default and can be used in other toolbars.
 
-##### Example:
+##### Example using quicktable in the editor toolbar
```

Comment:
For consistency, I think this should still have a colon at the end, as the other examples have a colon at the end.